bitcoin-review-podcast/lightning-privacy-splice-panel.md

```diff
@@ -1978,56 +1853,56 @@ If you're opening unannounced channels, that's one thing.
-And I guess we didn't really touch on this too much earlier, but the point of actually using a UTXO in Gossip is So it's a denial of service vector, I guess, if you didn't have that.
+And I guess we didn't really touch on this too much earlier, but the point of actually using a UTXO in Gossip is so it's a denial of service vector, I guess, if you didn't have that.
 So if you're freely able to gossip without having to actually lock Bitcoin into a channel, then the gossip network is basically unusable.
 So we still want that property.
 And so with Gossipy 2, there needs to be a way, it looks like zero knowledge proofs would be the idea to still say that you have Bitcoin locked up, but not saying exactly where it is and how much.
 
-Speaker 6: 01:37:30
+T-Bast: 01:37:30
 
 Yeah.
 I'd say that the state of that proposal is basically zero knowledge proofs, magic maths, details left as an exercise to the reader, but we'll see.
 
-Speaker 3: 01:37:40
+Vivek: 01:37:40
 
 So I've kind of declined in my LN whatever, keeping up with it.
 Gossip V2, correct me if I'm wrong, is Alex Meyer's proposal that has incorporated mini-sketch or is that 1.5? Okay.
 
-Speaker 6: 01:37:56
+T-Bast: 01:37:56
 
-Yeah, there are a lot of things about gossip that we need to fix because gossip is Basically the simplest form of gossip we could get away with that was easy to implement.
+Yeah, there are a lot of things about gossip that we need to fix because gossip is basically the simplest form of gossip we could get away with that was easy to implement.
 Nobody is ever satisfied with it, but it's really hard to make it good, to do something that's really good.
 So there are a lot of things we'd like to change, but some of these things conflict.
 Some of these things don't have a really nice solution for, for example, the ZKP stuff would really need a production grade ZKP library that would be available for everyone.
 
-Speaker 0: 01:38:25
+NVK: 01:38:25
 
-And
+And good luck with that
 
-Speaker 6: 01:38:26
+T-Bast: 01:38:26
 
 people still don't know what the system is gonna be the one that people will still care about in two years.
 So it's really-
 
-Speaker 1: 01:38:33
+Jeff: 01:38:33
 
 Yeah, we did an exploration a while back about that.
 And we just, you know, like you said, there just wasn't anything that was, I guess, thoroughly sort of gone through the wringer and tested enough that you'll be, I guess, happy with choosing.
 So still a lot of work.
 
-Speaker 0: 01:38:51
+NVK: 01:38:51
 
 Well, I mean, anytime you hear anything that requires a new implementation of a novel cryptographic primitive that is production grade, you add five years.
 
-Speaker 4: 01:39:07
+Dusty Damon: 01:39:07
 
 It's a good rule.
 
-Speaker 5: 01:39:08
+Tony: 01:39:08
 
 Yeah.
 Even Tapper channels themselves, they in order to broadcast them, you need a gossip chain.
```

Comment:
What is "Tapper channels"?
From the previous discussion is clear that here they are talking about "taproot channels"